src/core_write.cpp

```diff
@@ -69,7 +69,7 @@ std::string FormatScript(const CScript& script)
         ret += strprintf("0x%x ", HexStr(std::vector<uint8_t>(it2, script.end())));
         break;
     }
-    return ret.substr(0, ret.empty() ? ret.npos : ret.size() - 1);
+    return ret.substr(0, ret.empty() ? std::string::npos : ret.size() - 1);
```

Comment:
personally I prefer ret.npos, or at least think that either is fine and we don't need a linter for this

Reply:
dropped e16f6a34b5a200e3ab163dd935f26f17bd1c172f and rolled back to `ret.npos` instead of `std::string::npos` in ff9c66a11d5d2c4dc64d026a0719cb3fc586d17d

Reply:
I only left a comment on this line, but I meant all other lines as well.

I really don't get the recent rush to enforce meaningless style decisions with clang-tidy, even happily introducing bugs along the way (https://github.com/bitcoin/bitcoin/pull/25695#discussion_r954086171).

Reply:
Yes I meant I updated it everywhere as well.

Well, it came from a readability issue and so I thought it would be better to enforce it.

Reply:
Happy to drop ff9c66a11d5d2c4dc64d026a0719cb3fc586d17d and keep 6f8c291218041ae843dd9099c7103e607d0f8a39 if needed